.github/workflows/ci.yml

```diff
@@ -1,9 +1,8 @@
-# Copyright 2020 Signal Messenger, LLC
+# Copyright 2021 Signal Messenger, LLC
 # SPDX-License-Identifier: AGPL-3.0-only
 
 name: CI
-
-on: [push]
+on: [push, pull_request]
```

Comment:
Could you revert this part of the change?

Reply:
Of course, but could you explain why? It's normally best practice to run CI on PR to detect anything breaking or not formatted correctly before a merge.

Reply:
When a branch is pushed, it already triggers CI. The pull request trigger is redundant.

Reply:
By default GitHub Actions is disabled on forks, so this is only true when commits are pushed from the signalapp repo. Adding `pull_request` ensures CI runs on pull requests from all repositories.

Reply:
But if you still would like to have it removed, I can remove it of course